src/Machete.X12Schema/V5010/Layouts/LoopTIA_2_813.cs

```diff
@@ -0,0 +1,15 @@
+namespace Machete.X12Schema.V5010
```

Comment:
this should be deleted since LoopTIA_1_813 is exactly the same. If there is no TIA loop that is different then LoopTIA_1_813 should be renamed to LoopTIA_813